No wordexp(3) on OpenBSD.
XXX check for the wordexp function instead + push upstream

Index: src/libide/core/ide-global.c
--- src/libide/core/ide-global.c.orig
+++ src/libide/core/ide-global.c
@@ -28,7 +28,9 @@
 #include <sys/user.h>
 #include <sys/utsname.h>
 #include <unistd.h>
+#ifndef __OpenBSD__
 #include <wordexp.h>
+#endif
 
 #include <gio/gio.h>
 #define G_SETTINGS_ENABLE_BACKEND
@@ -399,6 +401,7 @@ _ide_host_environ (void)
 gchar *
 ide_path_expand (const gchar *path)
 {
+#ifndef __OpenBSD__
   wordexp_t state = { 0 };
   char *replace_home = NULL;
   char *ret = NULL;
@@ -441,6 +444,9 @@ ide_path_expand (const gchar *path)
   g_free (escaped);
 
   return ret;
+#else
+  return g_strdup (path);
+#endif
 }
 
 /**
